assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -212,9 +212,14 @@ che.docker.ip.external=NULL
 #   - 'docker-local': internal address is address of container within docker network, and exposed ports
 #                     are used.
 #   - 'custom': The evaluation strategy may be customized through a template property.
+#   - 'single-port':  internal address is set as in docker-local strategy, external address is composed
```

Comment:
Agreed. When we introduced the single port strategy the custom strategy wasn't implemented yet. And since then we haven't had the chance to rework it yet. But that's the direction we need to take.